go.mod

```diff
@@ -48,3 +49,5 @@ require (
 	gopkg.in/square/go-jose.v2 v2.4.1
 	sigs.k8s.io/testing_frameworks v0.1.2
 )
+
+replace github.com/dexidp/dex/api => ./api
```

Comment:
Is this necessary, what happens if this is removed? 

Looks like with this, we will have to revendor (if we are keeping that) every time this changes

Reply:
It is necessary. If it is removed, go mod will try to download it from GitHub.

I don't think the api changes that much, but vendoring is not used when building the final image, so no harm done if we don't revendor. :)